wrangler.toml

```
@@ -0,0 +1,31 @@
+# minibus wrangler config.
+name = "minibus"
+main = "./dist/worker.mjs"
+compatibility_date = "2022-05-11"
```

```suggestion
compatibility_date = "2022-05-11"
compatibility_flags = [ "url_standard" ]
```

this one is not handled by date just flag, but its important to properly handle url